application/src/main/data/json/tenant/rule_chains/root_rule_chain.json

```diff
@@ -18,7 +18,9 @@
         },
         "type": "org.thingsboard.rule.engine.telemetry.TbMsgTimeseriesNode",
         "name": "Save Timeseries",
-        "debugMode": false,
+        "debugFailures": false,
```

Comment:
Why in rule_chain_template we have 

```
        "debugFailures": false,
        "debugAll": false,
```

but here we have 

```
        "debugFailures": false,
        "debugAll": false,
        "debugAllUntil": 0,
```

? Any solution to skip all 3 parameters and use default values?

Reply:
Next time we add something, we will also modify all resources?

Reply:
According to the original design, we should have a 'debug' object instead of three fields. Absence of the object may mean that the debug is disabled. 

See https://github.com/thingsboard/thingsboard/pull/11861#issuecomment-2462606341